RELEASE-JMRI.md

```diff
@@ -48,13 +48,19 @@
   Another example:
   https://github.com/openlcb/OpenLCB_Java/commit/80793002a217d6f3cc2f188a532525d875652e0f
 
+7. Publish to Maven Central
+
+  ```mvn release```
+
 ## Update JMRI for new OpenLCB
 
 This process is generally defined by
 https://github.com/JMRI/JMRI/blob/master/lib/README.md#updates so it's worth to
 check out if the process changed form what's described below.
 
-1. Update the master branch of your fork by pulling from upstream
+1. Verify the version to update JMRI with is published at http://search.maven.org/#search|ga|1|openlcb
```

Comment:
The search interface of maven gets updates in about two hours, whereas the repository gets the published artifacts in about 10 minutes. Could you update this instruction to look into the repository instead of searching?